app/src/main/java/comcmput301f16t01/github/carrier/Notifications/NotificationController.java

```diff
@@ -26,7 +37,8 @@
         ElasticNotificationController.FindNotificationTask fnt = new ElasticNotificationController.FindNotificationTask();
```

Comment:
Sorry the above Javadoc I meant for here (even something simple like "Fetches sorted list of notifications from elasticsearch for a given user"